Add meta descriptions to Lisk-SDK, closes #371 (#387)
LiskHQ/lisk-docs: modules/ROOT/pages/lisk-elements/packages/client.adoc

= Lisk Client
Mona Bärenfänger <mona@lightcurve.io>

> :description: Technical reference for the Client package of Lisk Elements. It copmprises references to all included sub-packages and general usage instructions.

Reviewer: ```suggestion
:description: Technical references regarding the Clients packages of Lisk Elements. This consists of references to all included sub-packages and general usage instructions.
```
